dali/imgcodec/parsers/webp.cc

```diff
@@ -12,19 +12,118 @@
 // See the License for the specific language governing permissions and
```

Comment:
Please add at the top of the file the copyright from the `dali/image/webp.cc`, as this is based on it
```// Copyright (c) 2021, Lennart Behme. All rights reserved.```